packages/consoledot-api/src/queries/useSSOProvider.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from "@tanstack/react-query";
+import { useApiConfiguration } from "../ApiProvider";
+import { masQueries } from "../queryKeys";
+
+export type SsoProvider = {
+  tokenEndPointUrl: string;
+}
+
+export const useSSOProvider = () => {
+  const { basePath } = useApiConfiguration();
+
+  const { data: ssoProviders } = useQuery<SsoProvider>({
+    queryKey: masQueries.tokenEndPointUrl(),
+    queryFn: async () => {
+      const response = await fetch(
+        `${basePath}/api/kafkas_mgmt/v1/sso_providers`
+      );
+      const providers = await response.json();
+      const { token_url } = providers;
+      const ssoProvider: SsoProvider = { tokenEndPointUrl: token_url };
+      return ssoProvider;
+    },
+  });
+  const tokenEndPointUrl = ssoProviders?.tokenEndPointUrl;
+
+  if (!tokenEndPointUrl) {
```

Comment:
@riccardo-forina, In this line I am trying to check whether the tokenEndPointUrl is present or not and throwing the error, but it's making the application stop running and throw `Token URL is undefined`, could you please assist me with what I am doing wrong here